File: Programming EF Code First/chapter2/Chapter2/BAGA_FluentAPI/DataAccess/LodgingConfiguration.cs
using System.Data.Entity.ModelConfiguration;
using Model;

namespace DataAccess
{
  public class LodgingConfiguration : EntityTypeConfiguration<Lodging>
  {
    public LodgingConfiguration()
    {
      Property(l => l.Name)
        .IsRequired()
        .HasMaxLength(200);
    }
  }
}

File: Programming EF Code First/chapter3/Chapter 3 VB/Fluent/DataAccess/LodgingConfiguration.vb
Imports System.Data.Entity.ModelConfiguration
Imports Model

Namespace DataAccess
  Public Class LodgingConfiguration
	  Inherits EntityTypeConfiguration(Of Lodging)
	Public Sub New()
      Me.Property(Function(l) l.Name).IsRequired().HasMaxLength(200)

      Me.Property(Function(l) l.Owner).IsUnicode(False)

      Me.Property(Function(l) l.MilesFromNearestAirport).HasPrecision(8, 1)
	End Sub
  End Class
End Namespace